.github/workflows/nuget.yml

```diff
@@ -0,0 +1,69 @@
+name: Package nuget
+on:
+  push:
+    branches: [ "main", "feature/gh-nuget-workflow" ]
+  pull_request:
```

Comment:
Shouldn't we remove this? I don't think we want a push to Nuget when a PR is created, right?